tests/unit/models/gpflow/test_models.py

```diff
@@ -181,6 +181,33 @@ def test_gpr_raises_for_invalid_num_kernel_samples() -> None:
         GaussianProcessRegression(gpr_model(x, y), num_kernel_samples=-1)
 
 
+def test_gpr_raises_for_covariance_between_invalid_query_points_2() -> None:
```

Comment:
since we actually test the wrapper not the udnerlying GPR model

```suggestion
def test_gaussian_process_regression_raises_for_covariance_between_invalid_query_points_2() -> None:
```